sensor_msgs/Image image_initial
sensor_msgs/Image image_final
sensor_msgs/Imu[] last_imu